src/inertia.js

```diff
@@ -53,47 +53,57 @@ export default {
     this.cancelToken = axios.CancelToken.source()
 
     return axios({
-      method: method,
-      url: url,
+      method,
+      url,
       data: method.toLowerCase() === 'get' ? {} : data,
       params: method.toLowerCase() === 'get' ? data : {},
       cancelToken: this.cancelToken.token,
       headers: {
-        'Accept': 'text/html, application/xhtml+xml',
+        Accept: 'text/html, application/xhtml+xml',
         'X-Requested-With': 'XMLHttpRequest',
         'X-Inertia': true,
-        ...this.version ? { 'X-Inertia-Version': this.version } : {},
+        ...(this.version ? { 'X-Inertia-Version': this.version } : {}),
       },
-    }).then(response => {
```

Comment:
I did all these changes, other than the `.then()` indentation changes. Could you revert those? <3

Reply:
Just wanted to make the code a bit more clear (in my opinion) that's all 🙂

Reply:
Fair enough, but I think I prefer the previous formatting.

Reply:
Reverted 🙂

Reply:
Yeah I meant all of them, not just this one. :)

Reply:
Done 🙂